apps/site/src/components/navbar/account-dropdown.tsx

```diff
@@ -2,16 +2,17 @@ import {
   Box,
   Divider,
   ListItemButton,
+  // eslint-disable-next-line no-restricted-imports
```

Comment:
Result of changing ESLint rules. Importing Popover was restricted, but only from `"@mui/material/Popover`